modules/setting-up-vmc-for-vsphere.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vmc/installing-restricted-network-vmc-user-infra.adoc
+// * installing/installing_vmc/installing-vmc-customizations.adoc
+// * installing/installing_vmc/installing-vmc-network-customizations.adoc
+// * installing/installing_vmc/installing-vmc-network-customizations-user-infra.adoc
+// * installing/installing_vmc/installing-vmc-user-infra.adoc
+// * installing/installing_vmc/installing-vmc.adoc
+
+[id="setting-up-vmc-for-vsphere_{context}"]
+= Setting up VMC for vSphere
+
+You can install {product-title} on VMware Cloud (VMC) on AWS hosted vSphere clusters to enable applications to be deployed and managed both on-premises and off-premises, across the hybrid cloud.
+
+image::vmc-illustration.png["VMC on AWS Architecture"]
+
+You must configure several options in your VMware Cloud VMC environment prior to installing {product-title} on VMware vSphere. Ensure your VMC environment has the following prerequisites:
+
+* Create a non-exclusive, DHCP-enabled, NSX-T network segment and subnet. Other virtual machines (VM) can be hosted on the subnet, but at least eight IP addresses must be available for the {product-title} deployment.
+
+* Allocate two IP addresses, outside the DHCP range, and configure them with reverse DNS records.
+** A DNS record for `api.<cluster name>.<domain name>` pointing to the allocated IP address.
+** A DNS record for `*.apps.<cluster name>.<domain name>` pointing to the allocated IP address.
+
+* Configure the following firewall rules:
+** An ANY:ANY firewall rule between the {product-title} compute network and the Internet. This is used by nodes and applications to download container images.
+** An ANY:ANY firewall rule between the installation host and the software-defined data center (SDDC) management network. This allows you to upload the {op-system-first} OVA during deployment.
```

Comment:
I think this rule could be narrowed to only 443

![image](https://user-images.githubusercontent.com/5401402/111507312-e70ca180-8720-11eb-8469-50e7e998d5bd.png)

Reply:
@jcpowermac would this be correct?:

```suggestion
** An ANY:ANY firewall rule between the installation host and the software-defined data center (SDDC) management network on port 443. This allows you to upload the {op-system-first} OVA during deployment.
```